src/main/flight/pid.c

```diff
@@ -178,8 +178,8 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .launchControlAllowTriggerReset = true,
     );
 #ifdef USE_DYN_LPF
-    pidProfile->dterm_lowpass_hz = 120;
-    pidProfile->dterm_lowpass2_hz = 180;
+    pidProfile->dterm_lowpass_hz = 150;
+    pidProfile->dterm_lowpass2_hz = 150;
```

Comment:
Is this intended? Both dterm lpf with the same value? I suppose that being dynamic, maybe it does not matter but is strange to me at first.

Reply:
It is correct--right @ctzsnooze?  dlpf1 will increase with the throttle to 250 Hz to help reduce latency.

Reply:
Understood. But I'm thinking in the impression for the users. The MSP command for the dynamic lpf will be introduced, more or less, for version 4.1. So the user can not be aware of a new dynamic filter because there will not be any option in the configurator  (only CLI).

It can be enabled or not by default in version 4.0:
- If enabled, the user will see in the configurator two lpf in the same frequency, but the user will not be aware that one is modified dynamically.
-  If not enabled, then we are changing the values of the two lpf to the same value by default.

Maybe this is a temporary change, the dyn lpf will be disabled and the lpf will be restored to more standard values before the final release of 4.0.

Reply:
@ledvinap Would like a 4th order Butter be suitable for D?

Reply:
@kmitchel : My feeling is that 4th order is bit too high.
Doubling filter order will roughly double filter delay, doubling filter cutoff will reduce delay to 1/2.
Higher order filter has higher attenuation at frequencies far above cutoff (it has higher rolloff rate), but filter response close to cutoff is important FC performance.

There was a lot of discussion in past, with inconclusive results ...